### PR TITLE
Implement enough of Intro to get no loco running

### PR DESCRIPTION
**Pull request by duncanspumpkin**
I realised this was the reason i couldn't get gui to work when running without any calls.

### CODE REVIEW

File: src/OpenLoco/src/Intro.cpp [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
+#include "ViewportManager.h"
 #include <OpenLoco/Interop/Interop.hpp>
 
 using namespace OpenLoco::Interop;
 
 namespace OpenLoco::Intro
 {
+    loco_global<uint32_t, 0x0050C190> _50C190;
     loco_global<State, 0x0050C195> _state;
+    loco_global<bool, 0x0050C196> _50C196;
```
Comment (AaronVanGeffen) [inline src/OpenLoco/src/Intro.cpp:17]: These can be `static`, I believe.

File: src/OpenLoco/src/Intro.cpp [hunk 2]
```diff
@@ -25,6 +34,32 @@ namespace OpenLoco::Intro
     // 0x0046AE0C
     void update()
     {
+        addr<0x0005252E0, int32_t>() = 1;
+        if (_state == State::end)
+        {
+            Gfx::getDrawingEngine().getDrawingContext().clearSingle(Gfx::getScreenRT(), 0x0A);
+            if (!_50C196)
+            {
+                // Audio::stopIntro(); TODO: I think we might have broken audio in the intro...
```
Comment (AaronVanGeffen) [inline src/OpenLoco/src/Intro.cpp:43]: I don't think there was ever any audio in Locomotion's intro... Unlike RCT2, Locomotion just refers to the EULA until clicked.

Comment (duncanspumpkin) [reply src/OpenLoco/src/Intro.cpp:43]: yeah just realised that. Looks like this was the old RCT2 code.

**Comment by janisozaur** [comment]
This is getting interesting, I'll have to check it soon

**Review by AaronVanGeffen — APPROVED** [review]
Appears to be working fine. Both normally and with `--intro` passed in CLI.

**Pull request merged**